### DISCUSSION TITLE
what is the key code for gameshift key?

### DISCUSSION THREAD

**Discussion by ash-hashtag** [T-Troll/alienfx-tools#351] [Q&A]
I've tried a lot of keyboard event listeners, but none of them seem to pick up the gameshift key press.

I'm working to make this work on linux, and kinda got it working, right now it just toggles gmode, and now I'm trying to keybind it to the gameshift key, but can't find its key code. or is it a hardware key, which we can't use? how did you make it work on windows?

I'll probably make a pull request soon for other linux users too

**Comment by T-Troll** [comment]
It's F17 on Windows. BTW, Alien FX key is F18.
Hope Linux keyboard kmod support it.

You can see it into the source code....
